src/api/config/routes.rb

```diff
@@ -118,6 +118,7 @@ def self.matches?(request)
       post 'package/save_new_link/:project' => :save_new_link, constraints: cons
       post 'package/save/:project/:package' => :save, constraints: cons
       get 'package/delete_dialog/:project/:package' => :delete_dialog, constraints: cons
+      get 'package/trigger_services/:project/:package' => :trigger_services, constraints: cons
```

Comment:
It should... in the end the call that is performed to the backend is a POST too.